Responsive-desktop/styles.css

```diff
@@ -0,0 +1,550 @@
+  color: #fff;
+}
+
+.btn-in:disabled {
+  background-color: #fff;
+  border: 1px solid #c1c7d0;
+}
+
+.about {
+  display: flex;
+  flex-direction: column;
+  row-gap: 3em;
+  padding: 100px 24px;
+  background-color: #fff;
+  border-top-right-radius: 5em;
+}
+
+.about h2 {
+  font-size: 40px;
+  line-height: 52px;
+  font-weight: 800;
+  color: #172b4d;
+  width: 60%;
+}
+
+.about p {
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 24px;
+  color: #344563;
+}
+
+.user-tool-d {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.user-tool > div {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.user-tool > div > ul {
+  flex: 1 1 100%;
+  list-style: none;
+  margin-bottom: 10px;
+}
+
+.tool-list > li {
+  display: flex;
+  align-items: center;
+  column-gap: 20px;
+  height: 67px;
+  background-color: #f7f7f9;
+  margin-bottom: 12px;
+  border-radius: 8px;
+  font-size: 15px;
+  line-height: 20px;
+  color: #253858;
+  padding: 5px 12px;
+}
+
+hr {
+  height: 1px;
+  border: 0;
+  background-color: #dfe1e6;
+  width: 100%;
+}
+
+.contact-form {
+  background-color: #fff;
+}
+
+.contact-form h2 {
+  margin: 103px 24px 10px 24px;
+  color: #fff;
+  font-family: 'Poppins', sans-serif;
+  font-size: 40px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 52px;
+  letter-spacing: 0;
+  text-align: center;
+}
+
+.contact-form p {
+  margin: 12px 24px 30px 24px;
+  font-family: 'Poppins', sans-serif;
+  font-size: 20px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 28px;
+  letter-spacing: 0;
+  text-align: center;
+  color: #ebebff;
+}
+
+.contact-form-holder {
+  background-image: url('../img/contact-shape.png');
+  background-color: #6070ff;
+  background-repeat: no-repeat;
+  background-size: cover;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  border-top-left-radius: 5em;
+}
+
+.responsive-form {
+  width: 100%;
+  text-align: center;
+  max-width: 447px;
+}
+
+.ul-form {
+  list-style: none;
+}
+
+.li-form {
+  margin: 10px 24px;
+  text-align: left;
+}
+
+#name_form,
+#message,
+#email_form {
+  padding-top: 15px;
+  padding-bottom: 15px;
+  border-radius: 8px;
+  margin-bottom: 8px;
+  padding-left: 5px;
+  width: 100%;
+  border: none;
+  color: #172b4d;
+  font-family: 'poppins', sans-serif;
+  font-size: 17px;
+}
+
+#message {
+  height: 170px;
+}
+
+#get-submit {
+  width: 140px;
+  height: 48px;
+  margin-top: 10px;
+  margin-bottom: 45px;
+  color: #6070ff;
+  background-color: #fff;
+  box-shadow: 0 8px 16px rgba(64, 83, 252, 0.24);
+}
+
+#get-submit:hover {
+  background-color: #6070ff;
+  color: #fff;
+}
+
+#get-submit:active {
+  background-color: #2230d2;
+  color: #fff;
+}
+
+#get-submit:disabled {
+  background-color: #fff;
+  border: 1px solid #c1c7d0;
+}
+
+.logo:hover .profile-photo {
+  opacity: 1;
+}
+
+@media (min-width: 996px) {
```

Comment:
[HINT] You don't need to create a new `index.html` or `styles.css`. Just simply in the files of your mobile version, at the end of your `styles.css` add a `media` query, just like the one you have here. Now, start applying new styles in this block. Your mobile version will remain intact and all of the styles will reflect in screens with a bigger size than > `996px`.

[If you need more information you can refer to the documentation provided by Microverse on the subject.](https://github.com/microverseinc/curriculum-html-css/blob/main/responsivness.md)